withdrawn-packages.txt

```diff
@@ -10,3 +10,4 @@ rrdtool-utils-1.8.0-r0.apk
 ansible-operator-compat-1.36.1-r0.apk
 ansible-operator-1.36.1-r0.apk
 awx-operator-2.19.1-r0.apk
+filebeat-8.15.2-r0.apk
```

Comment:
we might want to remove all the packages here.

<details><summary>Details</summary>
<p>

```bash
$ wolfictl apk ls https://packages.wolfi.dev/os/x86_64/APKINDEX.tar.gz | rg filebeat
filebeat-8.12.0-r0.apk
filebeat-8.12.0-r1.apk
filebeat-8.12.0-r2.apk
filebeat-8.12.1-r0.apk
filebeat-8.12.1-r1.apk
filebeat-8.12.2-r0.apk
filebeat-8.12.2-r1.apk
filebeat-8.12.2-r2.apk
filebeat-8.13.0-r0.apk
filebeat-8.13.1-r0.apk
filebeat-8.13.1-r1.apk
filebeat-8.13.2-r0.apk
filebeat-8.13.2-r1.apk
filebeat-8.13.3-r0.apk
filebeat-8.13.3-r1.apk
filebeat-8.13.4-r0.apk
filebeat-8.13.4-r1.apk
filebeat-8.14.0-r0.apk
filebeat-8.14.0-r1.apk
filebeat-8.14.1-r0.apk
filebeat-8.14.1-r1.apk
filebeat-8.14.2-r0.apk
filebeat-8.14.3-r0.apk
filebeat-8.15.0-r0.apk
filebeat-8.15.0-r1.apk
filebeat-8.15.1-r0.apk
filebeat-8.15.1-r1.apk
filebeat-8.15.2-r0.apk
filebeat-8.15.3-r0.apk
filebeat-8.15.3-r1.apk
filebeat-8.15.4-r0.apk
filebeat-8.16.0-r0.apk
filebeat-8.16.1-r0.apk
```

</p>
</details>

Reply:
Thanks for the review @kranurag7!